client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -84,8 +84,18 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 			return;
 		}
 
-		// Check if the iframe is off the top of the screen and scroll back into view.
-		if ( 0 >= iframe.getBoundingClientRect().top ) {
+		/**
+		 * If the iframe is off the top of the screen
+		 * OR the iframe is off the bottom of the screen
+		 * scroll the window so the iframe is in view.
+		 */
+		if (
+			0 >= iframe.getBoundingClientRect().top ||
+			0 >=
+				window.innerHeight -
+					iframe.getBoundingClientRect().height +
```

Comment:
This should be a subtraction not an addition. Let's say we have a 460px high screen and 420px high modal and 210px top

the way it was written will result in 460-420+211 = 250 but it should actually be 460-420-210 = -164 or we could also do 460 - ( 420 + 211 ) = -164

Reply:
Ah, you're right. I think I was trying to move some parentheses around to appease the linter and accidentally removed a pair. I'll get that patched up.

Reply:
Fixed in [a1e0ac7](https://github.com/Automattic/woocommerce-payments/pull/4203/commits/a1e0ac7e37fba769f4e9c3cdcc8667c85a7f0ff9)